<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnsbAuthorLinkProperties</class>
 <widget class="QDialog" name="QnsbAuthorLinkProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Link Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbProperties">
     <property name="title">
      <string>General Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lbType">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbType">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Sequencial</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../images.qrc">
           <normaloff>:/icon/mininext</normaloff>:/icon/mininext</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Interactive</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../images.qrc">
           <normaloff>:/icon/miniint</normaloff>:/icon/miniint</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Adaptative</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../images.qrc">
           <normaloff>:/icon/miniadp</normaloff>:/icon/miniadp</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbTarget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Target:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbTarget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbInteractive">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Interactive Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lbITrigger">
        <property name="text">
         <string>Trigger:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbIKey">
        <property name="text">
         <string>Key:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QComboBox" name="cbIKey">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>INFO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>----------</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RED</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GREEN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>YELLOW</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BLUE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>----------</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CURSOR_UP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CURSOR_DOWN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CURSOR_LEFT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CURSOR_RIGHT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>----------</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ENTER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BACK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GUIDE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MENU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>POWER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EJECT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PLAY</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PAUSE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STOP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>REWIND</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RECORD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EXIT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>----------</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_UP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_DOWN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VOLUME_UP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VOLUME_DOWN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>----------</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>----------</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>E</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>H</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>I</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>J</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>N</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>O</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>R</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>V</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>W</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>----------</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>*</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>#</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbITrigger">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gbAdaptative">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Adaptative Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="lbATrigger">
        <property name="text">
         <string>Trigger:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QComboBox" name="cbAVariable">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbAVariable">
        <property name="text">
         <string>Variable:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbAOperator">
        <property name="text">
         <string>Operator:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cbAOperator">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>eq</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ne</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gte</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lte</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbAValue">
        <property name="text">
         <string>Value:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="leAValue">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbATrigger">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QnsbAuthorLinkProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QnsbAuthorLinkProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
